tests/regressions.rs

```diff
@@ -60,7 +60,6 @@ fn issue_1205() {
     let (mut poll, mut events) = init_with_poll();
 
     let waker = Arc::new(Waker::new(poll.registry(), WAKE_TOKEN).unwrap());
```

Comment:
`waker` doesn't need to be wrapped in an `Arc` anymore.

Reply:
I know but that was part of the regression test so I just left it.